tasks/01_containerized_ml_models/model_api_10/api.py

```diff
@@ -10,7 +10,7 @@
 
```

Comment:
I'd rather not modify anything in the `01_containerized_ml_models/` directory ://